app/views/editions/secondary_nav_tabs/_edit.html.erb

```diff
@@ -0,0 +1,127 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Edit",
+      heading_level: 2,
+      margin_bottom: 5,
+    } %>
+  </div>
+</div>
+
+<%= resource_form(@resource) do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Title",
+        },
+        id: "title",
+        name: "edition[title]",
+        value: @resource.title,
+        heading_size: "m",
+        error_items: errors_for(@edition.errors, "title".to_sym, use_full_message: false),
+      } %>
+
+      <%= render "govuk_publishing_components/components/textarea", {
+        label: {
+          heading_size: "m",
+          text: "Meta tag description",
+        },
+        name: "edition[overview]",
+        hint: "Some search engines will display this if they cannot find what they need in the main text",
+        value: @resource.overview,
+      } %>
+
+      <%= render "govuk_publishing_components/components/radio", {
+        heading: "Is this beta content?",
+        name: "edition[in_beta]",
+        inline: true,
+        heading_size: "m",
+        items: [
+          {
+            value: 1,
+            text: "Yes",
+            checked: @resource.in_beta,
+          },
+          {
+            value: 0,
+            text: "No",
+            checked: !@resource.in_beta,
+          },
+        ],
+      } %>
+
+      <%= render "govuk_publishing_components/components/textarea", {
+        label: {
+          heading_size: "m",
+          text: "Body",
+        },
+        name: "edition[body]",
+        value: @resource.body,
+        hint: ("Refer to #{link_to("Refer to the Govspeak guidance (opens in new tab)", "https://govspeak-preview.publishing.service.gov.uk/guide", target: "_blank", rel: "noopener", class: "govuk-link")}").html_safe,
+      } %>
+
+      <% if @resource.published_edition %>
+        <%= render "govuk_publishing_components/components/details", {
+          title: "Add a public change note",
+        } do %>
+          <%= render "govuk_publishing_components/components/radio", {
+            heading: "Add a public change note",
+            heading_level: 3,
+            heading_size: "m",
+            name: "edition[major_change]",
+            hint: "Telling users when published information has changed is important for transparency.",
+            items: [
+              {
+                hint_text: "A change note will be published on the page and emailed to users subscribed to email alerts. The ‘last updated’ date will change.",
+                text: "Yes - information has been added, updated or removed.",
+                checked: resource.major_change,
```

Comment:
Should we use `@resource` instead of `resource` (the latter seems to come from the `InheritedBase` gem, and we're using `@resource` in most places in this file—it would be good to be consistent.